capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -660,17 +661,21 @@ private void startHelpActivity() {
     }
 
     private void initOnlyQRScanning() {
-        if (isOnlyQRCodeScanning()) {
+        if (isOnlyQRCodeScanningEnabled()) {
```

Comment:
[Request] If qr code scanning is not enabled, then this should be skipped as well. Otherwise if I set only `setOnlyQRCodeScanning(true)` then it will change the UI, but QR code scanning won't work.

Reply:
oh okay

Reply:
If users do this configuration, then the qr code only mode should be ignored.
<img width="732" alt="Screenshot 2022-11-03 at 18 49 31" src="https://user-images.githubusercontent.com/13149498/199797209-8919d5b5-5d2b-4a30-ac4d-3da2eb0f3bd8.png">

Reply:
Done